src/parser.js

```diff
@@ -109,12 +111,9 @@ function tokenize(markdown) {
                 return ['h3', [line.substr(3).trim()]];
             }
 
-            if (line.startsWith('-')) {
-                return ['li', [line.substr(1).trim()]];
-            }
-
-            if (line.startsWith('*')) {
-                return ['li', [line.substr(1).trim()]];
+            const li = /\s*[-\*]\s+(.*)/.exec(line);
```

Comment:
This regex would match dashes in the middle of lines as well, i.e. `foo - bar`: https://regex101.com/r/hWmo7x/1 . It should probably start with a `^` to ensure it matches the start of the line.